src/universal/components/ResetPasswordPage/ResetPasswordForm.js

```diff
@@ -44,9 +44,14 @@ const PasswordResetForm = (props: Props) => {
           disabled={props.submitting}
         />
       </Block>
-      <PrimaryButton disabled={!props.valid} waiting={props.submitting} type="submit">
```

Comment:
I'm generally in favor of more components with fewer props to reduce component complexity (and the button component has a LOT of complexity!)

In the future, I'd love to break it down & start with a `PlainButton` that does nothing but remove html styles.
Then, we could build a RaisedButton on top of that, which adds hover & click transitions + a few other styles.
Then, we could have different styles of a RaisedButton that fill in the props for us.

That said, this LGTM. Personally, i still think it's weird that none of the buttons have box shadows, but maybe that's just me.

Reply:
👏 Yes to more components w/ fewer props! I generally also prefer things to be compositionally  over parametrically dynamic.

Reply:
I agree in that direction. In this case I was having to work on styles for state in 2 places, so it wasn’t really broken down in a way that is progressively more enhanced, it was doing the same thing (there were 2 ways to have a primary button and inconsistent styles between the 2).

I would be glad to capture an issue to refactor button variants.